Gems/ROS2/Code/Include/ROS2/Lidar/LidarRaycasterBus.h

```diff
@@ -101,6 +101,14 @@ namespace ROS2
         AZStd::vector<float> m_intensities;
     };
 
+    //! Structure used to describe both minimal and maximal
+    //! ray travel distance in meters.
+    struct Range
+    {
+        float m_min{ 0.0f };
+        float m_max{ 0.0f };
+    };
```

Comment:
I would optionally consider, if you do not want to rename this to e.g. RayRanges? In order to evade possible name collision if Range type will be added in some other place.

Reply:
Sure, that's a valid concer. I will change it to `RayRange` then.